packages/icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.98.1",
+  "version": "1.98.2",
```

Comment:
`1.99.0` would ideal here. You are not fixing anything here, so no need to increase patch version.

Reply:
Done